components/normalizer/benches/canonical_decomposition.rs

```diff
@@ -0,0 +1,43 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use criterion::{black_box, BenchmarkId, Criterion, Throughput};
+
+use icu_normalizer::properties::CanonicalDecomposition;
+
+fn function_under_bench(canonical_decomposer: &CanonicalDecomposition, character: char) {
+    canonical_decomposer.decompose(character);
+}
+
+pub fn criterion_benchmark(criterion: &mut Criterion) {
+    let group_name = "canonical_decomposition";
+    let decomposer = CanonicalDecomposition::try_new_unstable(&icu_testdata::unstable()).unwrap();
+    let params = black_box([
+        'ä',
+        'Ä',
+        'ệ',
+        'Ệ',
+        '\u{1D15E}',
+        'ো',
+        '𑄮',
+        '가',
+        '각',
+        '\u{212B}',
```

Comment:
U+212B, U+2126, U+1F71, U+1F72, and U+1D15E are unlikely to be performance-relevant in the real world. It's probably a bad idea to make such a large proportion of the benchmark about those. I suggest not benching these at all. Also, it would probably be relevant to include characters that don't decompose. That is, benching decomposing just plain ASCII a, for example.